src/pages/docs/service-level-agreement.md

```diff
@@ -0,0 +1,111 @@
+# Business Service Level Agreement
+
+## 1. web3.storage Service Level Agreement ("SLA").
+
+Nitro Data LLC ("Company") commits to provide a level of service for web3.storage Customers demonstrating:
+
+1.1 99.9% Uptime. The Upload Service will upload Customer Content at least 99.9% of the time and the Read Service will serve Customer Content at least 99.9% of the time, subject to Section 4.
+
+1.2 Penalties. If any of the Services fails to meet the above service level, the Customer will receive a Service Discount on their next monthly bill, as further described in Section 6 of this SLA.
+
+1.3 Upload Size Limit. Individual uploads to the Upload Service may have a size limit per API call, as described in the API documentation available at <https://web3.storage/docs/reference/http-api/>.
+
+1.4 Tier-Based Size Limit. Individual uploads are subject to size limits based on Customer’s chosen pricing tier as described in the Pricing page available at <https://web3.storage/account/payment/>.
+
+## 2. Definitions.
+
+2.1 "Claim" means a claim submitted by Customer to Company pursuant to this SLA that a Service Level has not been met and that a Service Discount may be due to Customer.
+
+2.2 "Customer" refers to the organization that has purchased and explicitly enabled Services from Company.
+
+2.3 "Customer Content" means any files, software, scripts, multimedia images, graphics, audio, video, text, data or other objects (a) available via a public IPFS gateway and requested for caching by Customer; (b) originating or transmitted from any device or website owned or operated by Customer and routed to, passed through and/or stored on or within the Company’s network or (c) otherwise transmitted or routed using the Services.
+
+2.4 "Customer Planned Downtime" is downtime specified by the Customer that is to be excluded from any calculation of an Outage Period. This would apply to any time when the Customer has requested access to Services suspended from their environment.
+
+2.5 "Force Majeure" refers to any downtime minutes that are the result of events or conditions beyond Company’s reasonable control. Such events might include but are not limited to any acts of common enemy, the elements, earthquakes, floods, fires, epidemics, and inability to secure products or services from other persons or entities.
+
+2.6 "Incident" means any set of circumstances resulting in a failure to meet a Service Level.
+
+2.7 "Outage Period" is equal to the number of downtime minutes resulting from an Unscheduled Service Outage.
+
+2.8 “Peered” means a persisted connection between peers configured on IPFS.
+
+2.9 "Services" means, collectively, the Upload Service and the Read Service.
+
+2.10 “Upload Service” refers to uploads through the HTTP API for file types and sizes supported by the API, as defined by the API docs available at <https://web3.storage/docs/reference/http-api/>.
```

Comment:
No link available

Reply:
@heyjay44 will see we should drop this point or rephrase to w3up

Reply:
2.10 “Upload Service” refers to uploads through the website, client, and CLI for supported file types and sizes as defined by the docs available at [https://web3.storage/docs/](https://web3.storage/docs/reference/http-api/).